nevergrad/benchmark/experiments.py

```diff
@@ -211,6 +211,24 @@ def oneshot(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                 yield Experiment(func, optim, budget=budget, num_workers=budget, seed=next(seedg))
 
 
+@registry.register
+def fiveshots(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    "One shot optimization of 3 classical objective functions (sphere, rastrigin, cigar)"""
+    seedg = create_seed_generator(seed)
+    names = ["sphere", "rastrigin", "cigar"]
+    optims = sorted(x for x, y in ng.optimizers.registry.items() if y.one_shot)
```

Comment:
one shots optimizer will behave exactly the same if you provide several workers: they do not adapt to the past evaluations, so this should lead to the same results than with `num_workers=budget`.
Are you looking for other optimizers?

Reply:
Done